Apps/WebClient/src/ClientApp/src/store/modules/patientData/getters.ts

```diff
@@ -0,0 +1,27 @@
+import { Dictionary } from "@/models/baseTypes";
+import PatientData from "@/models/patientData";
+import { LoadStatus } from "@/models/storeOperations";
+import {
+    PatientDataFileState,
+    PatientDataGetters,
+    PatientDataState,
+} from "@/store/modules/patientData/types";
+import { getPatientDataRecordState } from "@/store/modules/patientData/utils";
+
+export const getters: PatientDataGetters = {
+    patientDataFiles(
+        state: PatientDataState
+    ): Dictionary<PatientDataFileState> {
+        return state.patientDataFiles;
+    },
```

Comment:
I suspect it would be more useful to have getters similar to the ones for patientData/isPatientDataLoading that instead use the fileId to retrieve details of a particular file.

Reply:
I was following clinical docs which just returns it all, but I guess that's for a very different screen, I'll amend.